### PR TITLE
chore: initial sonar config

### PR DESCRIPTION
**Pull request by imrekb**

### CODE REVIEW

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
+sonar.projectKey=positive-js_mosaic
+sonar.organization=positive-js
+
+# Path to sources
+sonar.sources=.
+sonar.exclusions=docs/**,scripts/**,tests/**,tools/**
+sonar.inclusions=**/*.ts,**/*.js,**/*.html
+
+# Path to tests
+sonar.tests=packages/
```
Comment (Fost) [inline .sonarcloud.properties]: давай укажем пакеты 

- cdk
- mosaic
- mosaic-moment-adapter

**Comment by PositiveJS** [comment]
Preview docs changes for 9764acf at https://positive-js.github.io/mosaic-previews/pr434-9764acf/

**Comment by PositiveJS** [comment]
Preview docs changes for 1c889ea at https://positive-js.github.io/mosaic-previews/pr434-1c889ea/

**Comment by PositiveJS** [comment]
Preview docs changes for d31696f at https://positive-js.github.io/mosaic-previews/pr434-d31696f/

**Comment by PositiveJS** [comment]
Preview docs changes for 341cc21 at https://positive-js.github.io/mosaic-previews/pr434-341cc21/

**Comment by PositiveJS** [comment]
Preview docs changes for d067261 at https://positive-js.github.io/mosaic-previews/pr434-d067261/

**Comment by PositiveJS** [comment]
Preview docs changes for 0e17a9e at https://positive-js.github.io/mosaic-previews/pr434-0e17a9e/

**Comment by PositiveJS** [comment]
Preview docs changes for bff8c9d at https://positive-js.github.io/mosaic-previews/pr434-bff8c9d/

**Comment by PositiveJS** [comment]
Preview docs changes for 8fee5a8 at https://positive-js.github.io/mosaic-previews/pr434-8fee5a8/

**Comment by PositiveJS** [comment]
Preview docs changes for 05313e5 at https://positive-js.github.io/mosaic-previews/pr434-05313e5/

**Comment by PositiveJS** [comment]
Preview docs changes for 52fe9b1 at https://positive-js.github.io/mosaic-previews/pr434-52fe9b1/

**Pull request merged**